src/pages/Commits.tsx

```diff
@@ -1,6 +1,8 @@
+import { useEffect, useState } from "react";
 import { useQuery } from "@apollo/client";
 import styled from "@emotion/styled";
-import { useParams } from "react-router-dom";
+import queryString from "query-string";
```

Comment:
[nit] I believe it's preferred to use Spruce's query-string wrapper util, aka what you're doing [here](https://github.com/evergreen-ci/spruce/blob/b91133995a3b7af0676f1f609a5a0b4815298902/src/pages/commits/commitChart/ChartToggle.tsx#L8) :)